### ISSUE TITLE
Strange bugs and sound gliches with IGModAudioChannel and workshop

### ISSUE THREAD

**Issue by glebqip** [Facepunch/garrysmod-issues#3259]
### Details
BASS library very much (because we need positional sound). Our sounds were placed in addon folder and all be fine. When we sent all sounds to the workshop and started using it, we got random strange sound behavior (for example, it must play sounds / testsound / sound1 but we heard sounds / testsound / sound2 from the same folder).
Also after we uploaded all sounds to workshop, we got some random BASS BASS_ERROR_UNKNOWN errors when sound creates with errorcode 41.
### Steps to reproduce
Idk hou reproduce it, it's absolutely random.

**Comment by willox** [comment]
This might be related to #3210, but I can't reproduce it. Can you try again using the same sounds you tested this with originally on the [dev branch](https://wiki.garrysmod.com/page/Dev_Branch).

If it's still happening, post a link to the workshop addon containing the sounds if possible.

**Comment by glebqip** [comment]
Tested my script - still have random sound file misses.

**Comment by glebqip** [comment]
No, i moved to workshop version of addon and again it gives me BASS_ERROR_UNKNOWN and mechanical noise in random sounds with last dev branch update

**Comment by Grocel** [comment]
There is a new version of BASS. Supposedly it has fixed those BASS_ERROR_UNKNOWN problems with some file formats. (https://github.com/Facepunch/garrysmod-requests/issues/1071)

**Comment by Grocel** [comment]
To investigate this issue and #3401 further could also try to reproduce it with gm_bass3 installed (using the BASS3.PlayFile() function) and tell me if you get the same results? https://gmod.facepunch.com/f/gmodaddon/jina/gm-bass3-A-cross-plattform-shared-Lua-API-to-BASS/1/

In your error description you told us that it would just print sound1 but not sound2 or 3. Your example code can just print sound1 as the other both callback function do not have any print calls, Maybe you should adjust that code for better testing.

By the way I'm not sure if the above code does what you intended to do, because the timer.Create calls look like they would just cause a hell of a noise. I wouldn't be able to tell if everything sounds right or not. If you want to use a delay in each cycle you should just work with timer.simple calls inside the first timer.Create.
